scripts/demo_inference.py

```diff
@@ -73,6 +74,10 @@
 parser.add_argument('--pose_track', dest='pose_track',
                     help='track humans in video', action='store_true', default=False)
 
+
+"""----------------------------- Face options -----------------------------"""
+parser.add_argument('--face', type=int, default=False, help='show face detection or not')
```

Comment:
we can use the same format as pose_track, so we do not need to set 0 and 1

Reply:
Okay, I changed it.